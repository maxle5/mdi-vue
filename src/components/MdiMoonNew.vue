
<template>
  <svg :width="size" :height="size" :viewBox="viewBox">
      <path d="M12 20A8 8 0 1 1 20 12A8 8 0 0 1 12 20M12 2A10 10 0 1 0 22 12A10 10 0 0 0 12 2Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiMoonNew",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
      computed: {
        viewBox() {
          return `0 0 ${this.size} ${this.size}`;
        }
      }
  })
</script>
