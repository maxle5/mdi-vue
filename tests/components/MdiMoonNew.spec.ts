
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiMoonNew from "../../src/components/MdiMoonNew.vue";

test("MdiMoonNew snapshot", () => {
  const wrapper = mount(MdiMoonNew, {});
  expect(wrapper.html()).toMatchSnapshot();
});
