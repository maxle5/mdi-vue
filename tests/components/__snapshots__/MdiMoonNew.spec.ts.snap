// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiMoonNew snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M12 20A8 8 0 1 1 20 12A8 8 0 0 1 12 20M12 2A10 10 0 1 0 22 12A10 10 0 0 0 12 2Z" style="fill: currentColor;"></path>
</svg>"
`;
